bu-ssl.php

```diff
@@ -78,10 +78,61 @@ class SSL {
 		'override_url_scheme'       => true,
 		'content_security_policy'   => "default-src https: 'unsafe-inline' 'unsafe-eval'",
 		'csp_report_url'            => '',
-		// regex adopted from @imme_emosol https://mathiasbynens.be/demo/url-regex .
-		'http_img_regex'            => '@<img.*src\s{0,4}=.{0,4}(http:\/\/[^\s\/$.?#].[^\s\'"]*).+>@iS',
-		// see full list -- https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content#Types_of_mixed_content .
-		'http_all_regex'            => '@<(img|audio|video|object|iframe|script|link|iframe).*(?:src|data|href)\s{0,4}=.{0,4}(http:\/\/[^\s\/$.?#].[^\s\'"]*).+>@iS',
+	);
+
+	/**
+	 * The html tags that can have insecure content
```

Comment:
Should be html tags that may have insecure content. Can implies that they're allowed insecure content.